src/rules/noEmptyRule.ts

```diff
@@ -59,6 +71,16 @@ function walk(ctx: Lint.WalkContext<void>) {
     });
 }
 
+function isExcluded(node: ts.Node, options: Options): boolean {
+    const { parent } = node;
+    if (parent === undefined) {
+        return false;
+    }
+
+    return isExcludedConstructor(parent)
+        || (options.allowEmptyCatch === true && parent.kind === ts.SyntaxKind.CatchClause);
```

Comment:
you can simply rename `isExcludedConstructor` and add this condition to its return statement